src/Reflectivity/Process.extension.st

```diff
@@ -1,43 +1,8 @@
 Extension { #name : #Process }
 
-{ #category : #'*Reflectivity' }
-Process >> isActive: n [
-	<metaLinkOptions: #( + optionDisabledLink)>
-	^level = n or: [ level == nil ]
-]
-
-{ #category : #'*Reflectivity' }
-Process >> isMeta [
-	<metaLinkOptions: #( + optionDisabledLink)>
-	level ifNil: [ level := 0 ].
-	^level ~= 0
-]
-
-{ #category : #'*Reflectivity' }
-Process >> level [
-	<metaLinkOptions: #( + optionDisabledLink)>
-	^level ifNil: [ level := 0 ]
-]
-
 { #category : #'*Reflectivity' }
 Process >> rfeffectiveProcess [
```

Comment:
Why this method was not moved?

Reply:
It is used by Reflectivity, we do not need it for instrumentation

Reply:
and even there I can't find a user, I will add that to my TODO